index.d.ts

```diff
@@ -1,15 +1,16 @@
-import { Options as FocusTrapOptions } from 'focus-trap';
-import * as React from 'react';
-
-export = FocusTrap;
-
 declare namespace FocusTrap {
-  export interface Props extends React.AllHTMLAttributes<any> {
-    children: React.ReactElement<any>;
+  export interface FocusTrapProps {
```

Comment:
Thanks for filing this PR, @edewit. Is there a reason why you removed `children`? Maybe because it was incorrectly typed and should've been `ReactNode` instead? I just fixed that with #66